envs/environment-py39.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - pip
   - python=3.9
   - setuptools=52.0.0
+  - numpy=1.22
```

Comment:
Curious to know, what is the difference between putting numpy in this section, vs. the pip section? You don't want to specify a minor version like for all the other dependencies?

Reply:
The dependencies listed above will be installed using conda, while the ones in the pip section will use pip. If available, conda installation is more robust as it packages binaries and executables necessary for installation together with the Python library, but not all packages are available in conda.

Reply:
Hey @eugeneyarovoi , so initially I'd added above to conda install since numpy was required for pip wheel building prior to installs and saw failures for anything before `numpy==1.22.0`. I tested again recently and after updating all of the subsequent package versions and looks like it's fine, but if there's a preference for conda as the package handler re: @jeongyoonlee can leave as is. Did find a few other items breaking that I'd like to update quick for the arm64x build on mac so might have a quick fast-follow